contracts/HATVault.sol

```diff
@@ -91,16 +91,21 @@ contract HATVault is Claim, Deposits, Params, Withdrawals {
         HATVaultsRegistry _registry = HATVaultsRegistry(msg.sender);
         maxBounty = _params.maxBounty;
         bountySplit = _params.bountySplit;
-        (hatBountySplit.governanceHat, hatBountySplit.hackerHatVested) = _registry.hatBountySplit();
         committee = _params.committee;
         depositPause = _params.isPaused;
         registry = _registry;
         __ReentrancyGuard_init();
         _transferOwnership(_params.owner);
         tokenLockFactory = _registry.tokenLockFactory();
-        arbitrator = _registry.owner();
-        challengePeriod = 3 days;
-        challengeTimeOutPeriod = 5 weeks;
+
+        // Set vault to use default registry values where applicable
+        hatBountySplit = HATVaultsRegistry.HATBountySplit({
+            governanceHat: NULL_UINT,
+            hackerHatVested: 0
```

Comment:
no NULL_UINT here as well?

Reply:
Well it just saves gas but gets the same result. Since it's a struct I only check the governanceHat, so making this 0 is cheaper